src/engine/specification.md

```diff
@@ -140,7 +141,7 @@ This structure contains the attributes required to initiate a payload build proc
 
 #### Specification
 
-1. This method call maps on the `POS_FORKCHOICE_UPDATED` event of [EIP-3675](https://eips.ethereum.org/EIPS/eip-3675#specification) and **MUST** be processed according to the specification defined in the EIP.
+1. The values `(headBlockHash, finalizedBlockHash)` of this method call map on the `POS_FORKCHOICE_UPDATED` event of [EIP-3675](https://eips.ethereum.org/EIPS/eip-3675#specification) and **MUST** be processed according to the specification defined in the EIP.
```

Comment:
IMO, it would be good to have a notion of `safeBlockHash` in the EIP and the refer to it in the upstream JSON-RPC API changes. Here is the draft of the change: https://github.com/ethereum/EIPs/compare/master...mkalinin:eip3675-safe-head?expand=1

Reply:
I feel a bit mixed. This has nothing to do with consensus and *only* affects end user APIs which are not defined in core EIPs

Reply:
> This has nothing to do with consensus and *only* affects end user APIs which are not defined in core EIPs

True. Probably, the reason I am thinking this worth doing is *atomic* nature of fork choice state updates. Though, we may be explicit about the atomicity here, in the Engine API spec, it's a bit counter intuitive as atomic update specification is spread across two documents but should work

Reply:
done